Algolia.Search.Test/AlgoliaClientTest.cs

```diff
@@ -1135,6 +1135,24 @@ public void TestSaveAndGetRule()
 			AreEqualByJson(rule, _index.GetRule(ruleId));
 		}
 
+        [Fact]
+        public void TestTrySaveRuleWithEmptyObjectID()
+        {
+            ClearTest();
+
+            string ruleID = "";
+            JObject rule = generateRuleStub(ruleID);
+            try
+            {
+				_index.SaveRule(rule);
+                Assert.
```

Comment:
So it seems there is a typo here. The build on ci.appvoyer is giving 
```
AlgoliaClientTest.cs(1148,24): error CS1001: Identifier expected [C:\projects\algoliasearch-client-csharp\Algolia.Search.Test\Algolia.Search.Test.csproj]
AlgoliaClientTest.cs(1148,24): error CS1002: ; expected [C:\projects\algoliasearch-client-csharp\Algolia.Search.Test\Algolia.Search.Test.csproj]
```

Basically either remove the assert or put something valid

Reply:
Sorry about that, Visual Studio for Mac hasn't saved the file and I've committed without looking at the diff.

I've changed it a bit to make it use `Assert.Throws` like in the other tests.